cartoframes/examples.py

```diff
@@ -0,0 +1,381 @@
+"""Download, preview, and query example datasets for use in cartoframes
+examples. Try examples by `running the notebooks in binder
+<https://mybinder.org/v2/gh/CartoDB/cartoframes/master?filepath=examples>`__,
+or downloading them from the `cartoframes examples directory
+<https://github.com/CartoDB/cartoframes/tree/master/examples>`__.
+
+In addition to the functions listed below, this examples module provides a
+:py:class:`CartoContext <cartoframes.context.CartoContext>` that is
+authenticated against all public datasets in the https://cartoframes.carto.com
+account. This means that besides reading the datasets from CARTO, users can
+also create maps from these datasets.
```

Comment:
This will become a problem because of the limit of named maps. I'd explain that "maps created at this account will be periodically removed", and schedule a cleanup.